tutorials/index.md

```diff
@@ -223,7 +223,9 @@ The data is written to disk in the specified location (in this case `c:\\recordi
 
 Visualization of multimodal streaming data plays a central role in developing multimodal, integrative-AI applications. Visualization scenarios in \\psi are enabled by the __Platform for Situated Intelligence Studio__ (which we will refer to in short as PsiStudio)
 
-__Note__: Currently, PsiStudio runs only on Windows (although a future cross-platform version is planned - see more in our [Roadmap](/psi/Roadmap) document.) The tool is not currently shipped as an executable, so if you want to use it you will need to build the codebase; instructions for building the code are available [here](/psi/BuildingPsi). The tool is implemented by the `Microsoft.Psi.PsiStudio` project in the `Psi.sln` solution tree under `Sources\Tools\PsiStudio`. To run it, simply run this project after building it. To enable live visualization scenarios (more on that in a second), Platform for Situated Intelligence Studio is a COM server and registers itself as such with the Windows Registry the first time it is launched. You may receive a prompt to allow access for the tool to register itself on first launch.
+__Notes__:
```

Comment:
not sure why github shows so many new lines, preview looks fine in both vscode and here.